app/views.py

```diff
@@ -54,27 +54,21 @@ def handle_healthcheck():
 
 @app.route('/<appname>')
 def handle_app(appname):
-    verbose = False
-    chosen_region = None
-    query_string = request.query_string
-
-    if not query_string == "":
-        for query in query_string.split('&'):
-            if "verbose" in query.lower():
-                if query.endswith("1"):
-                    verbose = True
-            elif "region" in query.lower():
-                chosen_region = query[7:]
+
```

Comment:
I'm not sure why you're removing this? I think the idea with this is that you can make the request, supplying a query string such that it only displays one region, instead of all regions. Maybe it's broken? I don't think we should be just removing it.